examples/cloudwatch/main.tf

```diff
@@ -0,0 +1,42 @@
+# Make a certificate.
+resource "aws_acm_certificate" "default" {
+  domain_name = "default.${var.domain}"
```

Comment:
The domain name `default` is already used in the `default` scenario. Please modify this to `watch` or some other unique value.

Reply:
I have created a variable for it and set the default value to "watch".
This variable is used multiple times, including for the Vault deployment name. (see vault module variable "name")
Does it seem quirky to you to use "var.subdomain" here? (see main.tf of the example)